src/tests/cli/NewFwkLib/NewFwkLib.csproj.in

```diff
@@ -272,9 +272,6 @@
     <Compile Include="$(CMAKE_CURRENT_SOURCE_DIR)\NewTestObject\NoopAuthInit.cs">
       <Link>SmokePerf\NoopAuthInit.cs</Link>
     </Compile>
-    <Compile Include="$(CMAKE_CURRENT_SOURCE_DIR)\NewTestObject\PdxAutoSerializerObj.cs">
-      <Link>PdxTest\PdxAutoSerializerObj.cs</Link>
```

Comment:
Hmm.. besides the mistake that this referenced file isn't in the pull-request, I'm not sure this can go so easily.